### PR TITLE
Portfolio: desktop version

### PR DESCRIPTION
**Pull request by HenaryKhuraijam**
# Portfolio: desktop version
- Modified style.css

### CODE REVIEW

**Review by Favourezeugwa — CHANGES_REQUESTED** [review]
## STATUS: REQUIRES CHANGES 🚦 ⚠️ ♻️ 

Hi @HenaryKhuraijam  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


close lady

## Highlights 💯 
- No Linter errors ✔️
- Descriptive Pull request summary✔️
- Good commit messages ✔️
- Professional Readme file ✔️ 

## Required Changes ♻️
_Check the comments under the review._

- 🌸 I think it will be better if you increased the font size and spacing in your portfolio's header section so that it looks like figma. Currently, the font size seems to be a bit too tiny and the entire content too left aligned. 

Here is what  mean compared to Figma👇🏽 

| Your Headline section  |
| :---:   | 
|Screenshot 2023-02-24 at 21 40 29  |

|Figma's Headline section |
| :---:   | 
|Screenshot 2023-02-24 at 21 59 26  |


- 🌸 Also, can we reduce the width of your contact form? It is a bit too wide compared to what's on Figma .

This is what you currently have compared to Figma 👇🏽 

| Your contact-form  |
| :---:   | 
|Screenshot 2023-02-24 at 21 40 53  |

|Figma's contact form |
| :---:   | 
|Screenshot 2023-02-24 at 22 01 38  |



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by HenaryKhuraijam** [comment]
# Made changes  as suggested
- Modified in Headline and Contact form

File: index.html [hunk 1]
```diff
@@ -22,6 +22,7 @@
   </header>
```
Comment (pinkmoon25) [inline index.html:22]: - [ ] **[OPTIONAL] Good Job implementing the changes for the contact form :clap: ,However please make sure the hamburger icon is gone in the desktop version and the navbar is similar to what it is in figma.** 

**Your current Navbar design :point_down:** 
image

**Figma Navbar design :point_down:** 
image

**Review by pinkmoon25 — APPROVED** [review]
Hi @HenaryKhuraijam , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Linters :heavy_check_mark: 
GitHub flow :heavy_check_mark: 
Professional Readme :heavy_check_mark: 
Good Commits & PR :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**